src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
Changing the tests solves the compilation error with CS0618 issue. I don't see `#pragma warning disable 0618 / #pragma warning restore 0618` around any other Obsolete attributes in the solution. Is it safe to say that users can either handle the CS0618 warnings or will not run into them?